bitnami/airflow/templates/metrics/deployment.yaml

```diff
@@ -39,6 +39,13 @@ spec:
       nodeSelector: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.nodeSelector "context" $) | nindent 8 }}
       {{- end }}
       {{- end }}
+      {{- if or .Values.tolerations .Values.metrics.tolerations }}
+      {{- if .Values.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.tolerations "context" $) | nindent 8 }}
+      {{- else }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.tolerations "context" $) | nindent 8 }}
+      {{- end }}
+      {{- end }}
```

Comment:
According to the `values.yaml`, the `metrics.tolerations` parameter takes precedence over the global tolerations parameter.

> Note: this configuration supersedes the global tolerations configuration

So this logic should be the other way around: render `metrics.tolerations` if they are set, default to `tolerations` instead.

Reply:
Yes, I totally agree with that, but I used the same strategy as is used with nodeselector. Let me fix that then.

Reply:
Done. `metrics.tolerations` would be set if defined, if not `tolerations` would be set if defined.